plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -367,12 +392,15 @@ public void dropCorruptedTable(ConnectorSession session, SchemaTableName schemaT
     public void renameTable(ConnectorSession session, SchemaTableName from, SchemaTableName to)
     {
         try {
-            restSessionCatalog.renameTable(convert(session), toIdentifier(from), toIdentifier(to));
+            restSessionCatalog.renameTable(convert(session), toRemoteTable(session, from), toIdentifier(to));
```

Comment:
Could you add a test for renaming tables?

Reply:
thanks, added